ubuntu/standard/5.0/Dockerfile

```diff
@@ -13,6 +13,8 @@ FROM public.ecr.aws/ubuntu/ubuntu:20.04 AS core
 ARG DEBIAN_FRONTEND="noninteractive"
 
 # Install git, SSH, and other utilities
+RUN echo 'Acquire::http::Timeout "120";' >> /etc/apt/apt.conf
```

Comment:
Should there be `RUN echo 'Acquire::https::Timeout "120";' >> /etc/apt/apt.conf` as well?

Reply:
https://raspberrypi.stackexchange.com/questions/137478/how-to-increase-timeout-for-apt-update